src/reports.coffee

```diff
@@ -23,8 +23,8 @@ sendReports = (job, flag, done) ->
     from = moment().subtract(1, 'days').startOf('day').toDate()
     to = moment().subtract(1, 'days').endOf('day').toDate()
   else
-    from = moment().subtract(1, 'days').startOf('isoWeek').toDate()
-    to = moment().subtract(1, 'days').endOf('isoWeek').toDate()
+    from = moment().startOf('isoWeek').subtract(1, 'weeks').toDate()
+    to = moment().endOf('isoWeek').subtract(1, 'weeks').toDate()
```

Comment:
@napiergit I don't see how this makes a difference. As it was before it always runs on a Monday due to this - https://github.com/jembi/openhim-core-js/blob/master/config/default.json#L76

I do think your way is a better approach that is more fault tolerant. However, I don't think this would be causing the issue on momconnect unless someone changed the reporting cron schedule to a day other than Monday. Am I missing something?